app/user/views.py

```diff
@@ -166,15 +171,39 @@ def post(self, request, *args, **kwargs):
         except Exception as e:
             return Response({'message': str(e)}, status.HTTP_500_INTERNAL_SERVER_ERROR)
```

Comment:
implement this error handler as a global middleware

Reply:
I have implemented the error handler as a global middleware

Reply:
> I have implemented the error handler as a global middleware

Changes noted